tests/jeos/firstrun.pm

```diff
@@ -239,6 +239,16 @@ sub run {
 
     # release console and reattach to be used again as serial output
     if (is_s390x && is_svirt) {
+        # enable root ssh login, see poo#154309
+        if (is_alp || is_sle_micro('>=6.0')) {
```

Comment:
at some point we would need to include `is_sle_micro('>=6.0')` as part of the `is_alp` condition. But let's be careful and have it in a new PR. I can take care of it.